### PR TITLE
switched to swiperJS, removed flowbite from package.json and tailwind…

### PR DESCRIPTION
**Pull request by ritsiiitd**
… switched to swiperJS, removed flowbite from package.json and tailwind…

### CODE REVIEW

**Review by aryanteng — CHANGES_REQUESTED** [review]
Screenshot 2024-03-13 at 11 42 36 AM
The swiper looks good. Here are some changes:
1. Swiper Js arrows do not have our primary color, this can be done using custom CSS styles in the globals.css I think.
2. Swiper should be made infinite, meaning that after click next to the last photo I should come to the first.
3. The arrows should be outside of the image, not overlapping.

**Comment by ritsiiitd** [comment]
> Screenshot 2024-03-13 at 11 42 36 AM The swiper looks good. Here are some changes: 1. Swiper Js arrows do not have our primary color, this can be done using custom CSS styles in the globals.css I think. 2. Swiper should be made infinite, meaning that after click next to the last photo I should come to the first. 3. The arrows should be clearly distinct from the image, either make them thicker and white, or give our primary, whichever makes the contrast best.

Fixed

File: components/admissions/phd/CustomCarousel.js [hunk 1]
```diff
@@ -1,18 +1,91 @@
...
+          disableOnInteraction: false,
+        }}
+        pagination={{
+          clickable: true,
+        }}
+        navigation={{ nextEl: '.arrow-left', prevEl: '.arrow-right' }}
+        modules={[Autoplay, Navigation]}
+        className="mySwiper">
+        <SwiperSlide>
+          <div className=" h-80 flex justify-center items-center">
```
Comment (aryanteng) [inline components/admissions/phd/CustomCarousel.js:33]: please use a map function for this, do not repeat components

File: components/admissions/phd/CustomCarousel.js [hunk 2]
```diff
@@ -1,18 +1,91 @@
...
 
-import { Carousel } from 'flowbite-react';
+import { Autoplay, Pagination, Navigation } from 'swiper/modules';
+import {
+  ArrowBackIosNewRounded,
+  ArrowForwardIosRounded,
+} from '@mui/icons-material';
 
-export default function CustomCarousel() {
+export default function App() {
```
Comment (aryanteng) [inline components/admissions/phd/CustomCarousel.js:14]: why App?

**Review by aryanteng — CHANGES_REQUESTED** [review]

**Comment by aryanteng** [comment]
Screenshot 2024-03-16 at 7 43 58 PM
it's not round, it's oval, i think you need to fix the width and height equal and rounded-full for this to work.

**Comment by ritsiiitd** [comment]
> Screenshot 2024-03-16 at 7 43 58 PM it's not round, it's oval, i think you need to fix the width and height equal and rounded-full for this to work.

Updated✔

**Review by aryanteng — APPROVED** [review]

**Pull request merged**